docs/features/software-catalog/descriptor-format.md

```diff
@@ -700,6 +700,11 @@ sufficient to enter only the `metadata.name` field of that group.
 
 ### `spec.ancestors` [required]
```

Comment:
Note that i did not inform the reader here about them not actually being required anymore. Since we do not yet have versioned docs, it's not always clear what to state in the docs that does not lead the user to the wrong conclusions / actions. Before a release of these libraries has happened, we cannot reliably state that the field is optional and even should be left out.